### DISCUSSION TITLE
Data Migration (versions 3.3 and 4.10.1)

### DISCUSSION THREAD

**Discussion by phandja** [WeblateOrg/weblate#7146] [General]
Hi, I have completed the installation of weblate (4.10.1) via Docker and everything is working perfectly. On my old server I have the version 3.3. I would like to know if data migration between these two instances is compatible (My projects last invisible after copying them from the old server .../weblate/data to the new one .../volumes/weblate-docker_weblate-data/_data/).

**Comment by nijel** [comment]
You need to copy the database as well. Also, upgrade across major versions only works via 4.0.x. Firsts upgrade to that and then to the latest version...

**Reply by phandja** [reply]
I think it would be easy to install the 3.3 version via Docker and then do the pull for the upgrate. Where can I find the old docker images for those old versions? Thank you @nijel!

**Reply by nijel** [reply]
There are tags for each version. Just use we late/weblate:3.3-1 or something similar. Check tags on Docker hub or on GitHub repo to get the closest version.

**Reply by phandja** [reply]
Thanks @nijel. It seems to work. Last question, is there any other way to get logs besides the docker-compose logs command? (/var/log/ mapped with something)

**Reply by nijel** [reply]
The Docker container currently logs only to stdout, so the docker-compose logs is the way to get it (bug you can configure logging in Docker to do what you want, see https://docs.docker.com/config/containers/logging/configure/

**Reply by phandja** [reply]
Perfect! My 3.3 version is working o/. I tried to put some custumized settings in settings-override.py as I did for version 4.10.1 but it seems not working for 3.3. What is the correct way to do that for those old versions? Thank you once again!

**Reply by nijel** [reply]
I have no clue when these were introduced, sorry...
